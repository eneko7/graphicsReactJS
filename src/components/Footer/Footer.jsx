import React from 'react';
import style from './Footer.scss';
import Signature from '../Signature';

const Footer = () => (
  <footer className={style.footer}>
    <span className={style.footer_films}>FILMS</span>
    <span className={style.footer_copyright}>
      Copyright Â©
      {(new Date()).getFullYear()}
      FILMS.
      <Signature name="IHAR KARPUK" />
    </span>
  </footer>
);

export default Footer;
