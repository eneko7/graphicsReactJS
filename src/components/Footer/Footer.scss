.footer {
  width: 100%;

  .footer_films {
    font-size: 38px;
    color: #0aaee4;
    font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
    font-weight: 800;
    text-align: center;
    display: block;
    margin: 40px auto 40px auto;
  }

  .footer_copyright {
    font-size: 20px;
    color: #969797;
    font-family: Helvetica, Arial, Verdana, Tahoma, sans-serif;
    text-align: center;
    display: block;
    margin: 0 auto 40px auto;
  }
}
