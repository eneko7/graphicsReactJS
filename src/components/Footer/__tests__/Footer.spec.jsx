import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Footer from '../index';

describe('Footer', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Footer />);
  it('renders correctly', () => {
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
