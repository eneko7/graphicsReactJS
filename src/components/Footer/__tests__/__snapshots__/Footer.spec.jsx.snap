// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <span
    className="footer_films"
  >
    FILMS
  </span>
  <span
    className="footer_copyright"
  >
    Copyright Â©
    2018
    FILMS.
    <Signature
      name="IHAR KARPUK"
    />
  </span>
</footer>
`;
